js/cagov.core.js

```diff
@@ -16855,7 +16855,11 @@ $.extend($.expr.pseudos, {
 /* Accordion Bootstrap 4 */
 $(document).ready(function () {
     var accordion = $('.accordion').find('.collapsed'); 
+    var accordionTab = $('.accordion').find('a[data-toggle="collapse"]');
+    var accordioncarrot = $('<span aria-hidden="true"></span>');
```

Comment:
A _carrot_ is an orange (typ.) root vegetable (_Daucus carota_). Pretty sure this should have been _caret_ (an arrow-like symbol or character, "^").